rocketpy/plots/tank_plots.py

```diff
@@ -98,3 +150,10 @@ def all(self):
         -------
         None
         """
+        self.draw()
+        self.tank.fluid_mass.plot(*self.flux_time)
+        self.tank.net_mass_flow_rate.plot(*self.flux_time)
+        self.fluid_height()
+        self.fluid_volume()
+        self.fluid_center_of_mass()
```

Comment:
You should add an `.all_info` and `.info` method to the `Tanks` class and `Fluids` class now that the plots and prints are good